[2023-05-23T13:56:01.541+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.transform scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2023-05-23T13:56:01.547+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.transform scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2023-05-23T13:56:01.547+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T13:56:01.548+0000] {taskinstance.py:1363} INFO - Starting attempt 8 of 8
[2023-05-23T13:56:01.548+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T13:56:01.555+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): transform> on 2019-04-01 00:00:00+00:00
[2023-05-23T13:56:01.560+0000] {standard_task_runner.py:55} INFO - Started process 5835 to run task
[2023-05-23T13:56:01.564+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'transform', 'scheduled__2019-04-01T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmp21npyoo5']
[2023-05-23T13:56:01.566+0000] {standard_task_runner.py:83} INFO - Job 41: Subtask transform
[2023-05-23T13:56:01.577+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-23T13:56:01.615+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.transform scheduled__2019-04-01T00:00:00+00:00 [running]> on host c3034470bbd8
[2023-05-23T13:56:01.659+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2019-04-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=8
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-01T00:00:00+00:00
[2023-05-23T13:56:01.660+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-23T13:56:01.677+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 188, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/algolia-dag.py", line 44, in transform
    filter_data.assign(has_specific_prefix=[True if x.index_prefix != 'shopify_' else False for x in filter_data['index_prefix']])
  File "/opt/airflow/dags/algolia-dag.py", line 44, in <listcomp>
    filter_data.assign(has_specific_prefix=[True if x.index_prefix != 'shopify_' else False for x in filter_data['index_prefix']])
AttributeError: 'str' object has no attribute 'index_prefix'
[2023-05-23T13:56:01.686+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=algolia_test, task_id=transform, execution_date=20190401T000000, start_date=20230523T135601, end_date=20230523T135601
[2023-05-23T13:56:01.692+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 41 for task transform ('str' object has no attribute 'index_prefix'; 5835)
[2023-05-23T13:56:01.737+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-05-23T13:56:01.751+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
